Library/OcCpuLib/OcCpuLib.c

```diff
@@ -1363,13 +1375,16 @@ OcCpuScanProcessor (
     Cpu->ExtFamily = (UINT8) Cpu->CpuidVerEax.Bits.ExtendedFamilyId;
     Cpu->Brand     = (UINT8) Cpu->CpuidVerEbx.Bits.BrandIndex;
     Cpu->Features  = (((UINT64) Cpu->CpuidVerEcx.Uint32) << 32ULL) | Cpu->CpuidVerEdx.Uint32;
+    Cpu->Hypervisor = (BOOLEAN) Cpu->CpuidVerEcx.Bits.HYPERVISOR;
```

Comment:
This will take time to get merged upstream. I would rather not apply changes to audk on my own, see my comment in https://github.com/acidanthera/audk/pull/2. For now it is best to use `NotUsed` with a corresponding bug link.